webbeans-junit5/pom.xml

```diff
@@ -76,17 +76,9 @@
                     <execution>
                         <id>default-test</id>
                         <configuration>
-                            <skip>true</skip>
-                        </configuration>
-                    </execution>
-                    <execution>
-                        <id>perclass</id>
-                        <phase>test</phase>
-                        <goals>
-                            <goal>test</goal>
-                        </goals>
-                        <configuration>
-                            <includes>**/perclass/*</includes>
+                            <excludes>
+                                <exclude>**/reusable/*</exclude>
+                            </excludes>
```

Comment:
almost unrelated but otherwise the new test would have not been picked up by surefire